pkg/stub/handler.go

```diff
@@ -304,6 +305,52 @@ func (h *Handler) updateCfgArch(cfg *v1.Config) *v1.Config {
 	return cfg
 }
 
+func (h *Handler) tbrInaccessible() bool {
+	if h.configclient == nil {
+		// unit test environment
+		return false
+	}
+	// we still check for this in case we have proxies in a ipv6 env;
+	// if proxies will not be supported in ipv6 single stack, we can
+	// remove this
+	if util.IPv6Only() {
+		return true
+	}
+	err := wait.PollImmediate(5*time.Second, 3*time.Minute, func() (bool, error) {
+		// if not set, unit test environment, so don't bother
+		proxy, err := h.configclient.Proxies().Get("cluster", metav1.GetOptions{})
+		if err == nil &&
+			(len(proxy.Status.HTTPProxy) > 0 || len(proxy.Status.HTTPSProxy) > 0) {
+			// if proxies are involved we assume we are not in a disconnected scenario
+			return true, nil
+		}
```

Comment:
So the assumption here is that if the cluster has a global proxy, it should allow connections to registry.redhat.io.

Reply:
correct ... it was a conclusion @bparees and I reached in https://github.com/openshift/cluster-samples-operator/pull/214#discussion_r364829755

Reply:
that said ... if a proxy is in play the TBR connection test below should pass as well ... I'll remove